Cargo.toml

```diff
@@ -15,11 +15,12 @@ rustc_version = "0.2"
 rustc-hex = { version = "1.0", optional = true }
 heapsize = { version = "0.4", optional = true }
 byteorder = { version = "1", default-features = false }
+crunchy = "0.1.5"
```

Comment:
your crunchy dependency introduced must have non-std variant

did you try to compile bigint with `--no-default-features` ?